src/fellows.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { ApiPromise, WsProvider } from "@polkadot/api";
+
+import { ActionLogger } from "./github/types";
+
+type FellowData = { address: string; rank: number };
+
+export const fetchAllFellows = async (
```

Comment:
This functionality was copied from [`review-bot/fellows.ts`](https://github.com/paritytech/review-bot/blob/main/src/polkadot/fellows.ts)